### PR TITLE
feat(components/molecule/tabs): Add scroll into active tab

### PR DESCRIPTION
**Pull request by stivaliserna**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### Description, Motivation and Context

This PR is adding scroll into active tab as default behaviour. 
### Screenshots - Animations

https://user-images.githubusercontent.com/6487058/134882881-a60a282f-aee2-4f24-94b8-27f5ae6d576f.mov

### CODE REVIEW

File: components/molecule/tabs/src/components/MoleculeTab.js [hunk 1]
```diff
@@ -20,16 +22,28 @@ const MoleculeTab = ({
   label,
   numTab
 }) => {
+  const [isIntersecting, outerRef] = useNearScreen({offset: '0px'})
```
Comment (andresin87) [inline components/molecule/tabs/src/components/MoleculeTab.js]: `useNearScreen` is a wrapper of useOnScreen with default offset set to 200px. if you want to set if to 0px, better use `useOnScreen` directly which its default offset value is 0px by default.

Take a look on https://github.com/SUI-Components/adevinta-spain-components/blob/master/components/react/hooks/src/useOnScreen/index.js

**Review by andresin87 — CHANGES_REQUESTED** [review]
change `useNearScreen({offset: '200px'})` for `useOnScreen()`

File: components/molecule/tabs/src/components/MoleculeTab.js (hunk 1 above)
Comment (stivaliserna) [reply components/molecule/tabs/src/components/MoleculeTab.js]: Nice! 👌 I took a quick look and didn't notice it! Thanks!

**Comment by stivaliserna** [comment]
> change `useNearScreen({offset: '200px'})` for `useOnScreen()`

Updates done! Thanks @andresin87! 🙌

**Review by Sabri1209 — APPROVED** [review]

**Review by arnau-rius — APPROVED** [review]

**Review by andresin87 — APPROVED** [review]

**Pull request merged**